scripts/sanitycheck

```diff
@@ -2341,15 +2341,28 @@ def main():
                     COLOR_YELLOW,
                     COLOR_NORMAL,
                     reason))
+    from copy import deepcopy
 
-    ts.instances = OrderedDict(
+    tmp = OrderedDict(
         sorted(ts.instances.items(), key=lambda t: t[0]))
```

Comment:
Additionally, we call sorted() here, why not try to sort "native_posix" to the top right here too?

Reply:
This can be done like so:

```Python
from functools import cmp_to_key

def native_posix_first(a, b):
    if a == 'native_posix':
        return -1
    return (a > b) - (a < b)

tmp = OrderedDict(sorted(ts.instances.items(), key=cmp_to_key(native_posix_first)))
```